.github/workflows/release-api-gateway-2.yaml

```
@@ -0,0 +1,179 @@
+name: Release Api Gateway - Step 2
+
+on:
+  push:
+    branches:
+      - 'release-**'
+    paths:
+      - 'sec-scanners-config.yaml'
+
+jobs:
+  check-prerequisitese: 
+    name: check-prerequisites
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get current release from sec-scanners-config.yaml file
+        shell: bash
+        run: |
+            CURRENT=$(cat sec-scanners-config.yaml | grep europe-docker.pkg.dev/kyma-project/prod/api-gateway/releases/api-gateway-manager | cut -d ":" -f2
```

This grep statement is not in sync with how we do the sed in the first step
```
sed -i "s|.*api-gateway-manager:.*|$IMAGE\:$IMAGE_TAG|g" sec-scanners-config.yaml
```
The above does not expect `IMAGE==europe-docker.pkg.dev/kyma-project/prod/api-gateway/releases/api-gateway-manager` while this grep does.
(there's also a missing closing bracket)
```suggestion
            CURRENT=$(cat sec-scanners-config.yaml | grep europe-docker.pkg.dev/kyma-project/prod/api-gateway/releases/api-gateway-manager | cut -d ":" -f2)
```